tests/foreman/cli/test_oscap.py

```diff
@@ -24,28 +35,41 @@
     tier4,
     skip_if_bug_open,
 )
-from robottelo.datafactory import (
-    filtered_datapoint,
-    valid_data_list
-)
 from robottelo.test import CLITestCase
 
 
 @filtered_datapoint
-def valid_scapcontent_list():
-    """Returns a list of valid environment names"""
-    return[
+def invalid_id_list():
+    """Returns a list of invalid ids"""
+    return [
         gen_string('alpha'),
-        gen_string('numeric'),
-        gen_string('alphanumeric'),
-        gen_string('alphanumeric', 255)
+        gen_string('alphanumeric')
```

Comment:
Could you please describe these changes? Numeric values aren't supported anymore? 255 chars is not valid boundary value? What's the new one then?
Btw, pls also think of moving this helper to datafactory, i'm not sure why it's not there yet.

Reply:
@abalakh these are invalid ID , which are supposed to be numeric